gradle.properties

```diff
@@ -17,8 +17,8 @@
 # http://www.gradle.org/docs/current/userguide/multi_project_builds.html#sec:decoupled_projects
 # org.gradle.parallel=true
 
-VERSION_CODE=9
-VERSION_NAME=1.3.0-SNAPSHOT
+VERSION_CODE=10
+VERSION_NAME=1.3.0
```

Comment:
you shouldn't change the `VERSION_NAME`. This should be always a SNAPSHOT release, since Travis auto uploads after every successful build of master. Just keep it 1.3.0-SNAPSHOT, we change it locally to 1.3.0 for the release and then update to 1.4.0-SNAPSHOT afterwards (and commit).